app/controllers/employments/EndEmploymentController.scala

```diff
@@ -222,7 +221,9 @@ class EndEmploymentController @Inject()(
             IrregularPayForm.createForm,
             EmploymentViewModel(mandatoryJourneyValues(0), mandatoryJourneyValues(1).toInt)))
       }
-      .getOrElse(throw new RuntimeException("Could not retrieve mandatory journey values"))
+      .getOrElse {
+        NotFound(errorPagesHandler.error5xx("Could not retrieve mandatory journey values"))
```

Comment:
did we decide we are returning a NotFound in all cases? if so should we be using errorPage4xx